<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVFloodFillEmbeddedWidget</class>
 <widget class="QWidget" name="CVFloodFillEmbeddedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="mpLowerTitleLabel">
     <property name="text">
      <string>Lower Differences</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="mpUpperTitleLabel">
     <property name="text">
      <string>Upper Differences</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mpLowerBLabel">
     <property name="text">
      <string>B value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="mpLowerBSpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="mpUpperBLabel">
     <property name="text">
      <string>B value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="mpUpperBSpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mpLowerGLabel">
     <property name="text">
      <string>G value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="mpLowerGSpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="mpUpperGLabel">
     <property name="text">
      <string>G value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="mpUpperGSpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mpLowerRLabel">
     <property name="text">
      <string>R value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="mpLowerRSpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="mpUpperRLabel">
     <property name="text">
      <string>R value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="mpUpperRSpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mpLowerGrayLabel">
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="mpLowerGraySpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="mpUpperGrayLabel">
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="mpUpperGraySpinbox">
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="mpMaskTitleLabel">
     <property name="text">
      <string>Mask  :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="mpMaskStatusLabel">
     <property name="text">
      <string>Inactive</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
